src/natcap/invest/sdr/sdr.py

```diff
@@ -108,62 +100,49 @@
                     "about": "Support practice factor for the USLE"}
             },
             "about": (
-                "A CSV table containing model information corresponding to "
-                "each of the land use classes in the LULC raster input."),
+                "A table mapping each LULC code to biophysical properties of "
+                "that LULC class. All values in the LULC raster must have "
+                "corresponding entries in this table."),
             "name": "Biophysical Table"
         },
         "threshold_flow_accumulation": {
-            "expression": "value > 0",
-            "type": "number",
-            "units": u.pixel,
-            "about": (
-                "The number of upstream cells that must flow into a cell "
-                "before it's considered part of a stream such that retention "
-                "stops and the remaining export is exported to the stream. "
-                "Used to define streams from the DEM."),
-            "name": "Threshold Flow Accumulation"
+            **spec_utils.THRESHOLD_FLOW_ACCUMULATION
         },
         "k_param": {
             "type": "number",
             "units": u.none,
             "about": "Borselli k parameter.",
-            "name": "Borselli k Parameter"
+            "name": "Borselli k parameter"
         },
         "sdr_max": {
             "type": "ratio",
-            "about": "Maximum SDR value.",
-            "name": "Max SDR Value"
+            "about": "The maximum SDR value that a pixel can have.",
+            "name": "maximum SDR value"
         },
         "ic_0_param": {
             "type": "number",
             "units": u.none,
             "about": "Borselli IC0 parameter.",
-            "name": "Borselli IC0 Parameter"
+            "name": "Borselli IC0 parameter"
         },
         "l_max": {
             "type": "number",
             "expression": "value > 0",
             "units": u.none,
             "about": (
-                "Values of L (the slope length component of the LS "
-                "slope length * slope gradient factor) larger than this value "
-                "will be clamped to this value. Ranges of 122-333 (unitless) "
-                "are found in relevant literature such as "
-                "Desmet and Govers, 1996 and Renard et al., 1997 "
-                "(see user's guide)."),
-            "name": "Max L Value",
+                "The maximum allowed value of the L parameter in the LS "
+                "factor."),
```

Comment:
Would it be helpful to include what `l / L` is in this case? Maybe: "... value of the slope length (L) parameter"